src/main/java/core/basesyntax/service/impl/CsvFileReaderService.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileReaderService implements FileReaderService {
+    @Override
+    public List<String> read() {
+        String filePath = "./src/main/resources/input-file.csv";
```

Comment:
pass path to file as a parameter of method below to make this service more flexible